src/main/kotlin/Archives.kt

```diff
@@ -0,0 +1,54 @@
+class ArchiveMenu (private val archives: MutableList<Archive>) : AbstractMenu(){
+
+    override fun show() {
+
+        while(true) {
+            println("\nСоздание архива: ")
+            println("0. Вернуться назад")
+
+            val name = avoidEmptyFormat("Введите название архива или 0 для возвращения: ")
+            if (name == "0") return
+            archives.add(Archive(name))
+            return
+        }
+    }
+
+}
+
+class ArciveList (private val archives: List<Archive>) : AbstractMenu(){
```

Comment:
Тут опечатка небольшая:
```suggestion
class ArchiveList (private val archives: List<Archive>) : AbstractMenu(){
```